src/app/pages/jobs/jobs.component.html

```diff
@@ -0,0 +1,40 @@
+All of your Jobs <br><br>
+
+<div class="current"> Current Jobs </div> <br>
+<div class="cust-cards">
+    <p class="title"> My speaker is broken </p>
```

Comment:
If this truly is a title for a section it should use one of the various header tags. `<h3>` por ejemplo